src/app/give-help/offer-form/offer-form.component.html

```diff
@@ -2,12 +2,14 @@
   <div class="container">
     <div class="row">
       <div class="col-md-12">
-        <label><b>Tytuł ogłoszenia </b> <span>(max 80 znaków): </span></label>
+        <label
+          ><b>{{ "LABEL_OFFER_TITLE" | translate }}</b> <span> {{ "LABEL_OFFER_MAXCHARS" | translate }} *</span></label
```

Comment:
prettier breaking tags in a strange way?

Reply:
fakt, puszczałem npm run format spróbuję coś zaradzić